src/EventBusRabbitMQ/RabbitMQEventBus.cs

```diff
@@ -200,16 +200,8 @@ private async Task ProcessEvent(string eventName, string message)
         // REVIEW: This could be done in parallel
 
         // Get all the handlers using the event type as the key
-        // foreach (var handler in scope.ServiceProvider.GetKeyedServices<IIntegrationEventHandler>(eventType))
-
-        _subscriptionInfo.HandlerTypes.TryGetValue(eventType, out var handlerTypes);
```

Comment:
Need this undeleted as well.

Reply:
Does the keyed services not replace the whole `HandlerTypes` business? That you don't need to keep track of all the handler types to resolve, you rely on the container instead.

Reply:
Oh you're right here.